src/test/java/org/jabref/logic/cleanup/RenamePdfCleanupTest.java

```diff
@@ -51,31 +54,36 @@ public void setUp() throws Exception {
      * Test for #466
      */
     @Test
-    public void cleanupRenamePdfRenamesFileEvenIfOnlyDifferenceIsCase() throws IOException {
+    public void cleanupRenamePdfRenamesFileEvenIfOnlyDifferenceIsCase(@TempDirectory.TempDir Path testFolder) throws IOException {
         String fileNamePattern = "[bibtexkey]";
-        File tempFile = testFolder.newFile("toot.tmp");
+        Path path = testFolder.resolve("toot.tmp");
+        Files.createFile(path);
+        File tempFile = path.toFile();
+
         LinkedFile fileField = new LinkedFile("", tempFile.getAbsolutePath(), "");
```

Comment:
The path class also has a method toAbsolutePath which you can use here., e.g. just toAbsolutePath().toString()